platform/podjs/src/browserPod.ts

```diff
@@ -297,23 +306,9 @@ class IDBPolyOut implements PolyOut {
                     );
                 },
             };
+        } else {
+            throw new Error(`Could not identify file for ${id}`);
         }
-
-        const file = await this.getFileInfo(id);
-        return {
-            async read() {
-                return new Uint8Array(await file.blob.arrayBuffer());
-            },
-            stat() {
-                return new CompatStats(
-                    id,
-                    file.blob.size,
-                    file.time.toISOString(),
-                    file.name,
-                    false
-                );
-            },
-        };
```

Comment:
🤔 I might be mistaken about this. I will check it tomorrow.

Reply:
Apparently that is another reason why the `google` and `facebookImport` feature is broken in *pod.js* since this change landed. So yeah, it wasn't dead code.